src/main/java/seedu/address/model/person/Person.java

```diff
@@ -21,21 +22,29 @@ public class Person {
     private final Email email;
 
     // Data fields
-    private final Address address;
-    private final HospitalWing hospitalWing;
-    private final Set<Tag> tags = new HashSet<>();
+    private final NextOfKin nextOfKin;
+    private final PatientType patientType;
+    private final Optional<HospitalWing> hospitalWing;
+    private final Optional<FloorNumber> floorNumber;
+    private final Optional<WardNumber> wardNumber;
```

Comment:
Great use of `Optional` for the case of outpatients not having these displayed in their `PersonCard`!

Reply:
Great job implementing add, thanks for the effort! I think we can merge this.